prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1720,7 +1721,7 @@ int rxr_pkt_proc_write_rta(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 	hdr_size = rxr_pkt_req_hdr_size(pkt_entry);
 	data = (char *)pkt_entry->pkt + hdr_size;
 	iov_count = rta_hdr->rma_iov_count;
-	rxr_rma_verified_copy_iov(ep, rta_hdr->rma_iov, iov_count, FI_REMOTE_WRITE, iov);
+	rxr_rma_verified_copy_iov(ep, rta_hdr->rma_iov, iov_count, FI_REMOTE_WRITE, iov, desc);
```

Comment:
Why not just pass NULL instead of an empty desc? At a minimum you need to initialize desc.

Reply:
The function ```rxr_rma_verified_copy_iov``` expect ```desc``` to be not NULL, and am I am hesitant to make it be able to handle NULL because that would add another branching.

Also because ```rxr_rma_verified_copy_iov``` will be set content of desc, I did not initialize it.

Reply:
Now I'm even more confused. Why is `rxr_rma_verified_copy_iov` initializing it?

Reply:
```rma_iov``` contain a key. For that key, the function ```rxr_rma_verified_copy_iov``` will find its corresponding MR and put the MR in ```desc```.

Reply:
I understand now, thank you.